02-deploying-an-app/002-app-deploy.md

```diff
@@ -49,7 +49,7 @@ Select your newly created repository from the displayed list.
 
 Within the repository, you will see the following section with values unique to you:
 
-![Image](images/repo-values.png)
+![Image](/images/repo-values.png)
```

Comment:
The first part of this (`[Image]`) is the alt text for the image. We should either leave it empty, or add an appropriate description (ideally the latter, because I’m sure we’ll have developers join us with visual impairments).

Reply:
Yeah I noticed this too, was going to do this in a separate PR as there are a few things like this to fix.